### ISSUE TITLE
Unable to change link font

### ISSUE THREAD

**Issue by bannmann** [asciidoctor/asciidoctor-pdf#575]
I am unable to change the font family for links. The `font_family` property does not seem to have any effect. Even when I specify an invalid font family, no error message is displayed. However, that happens when I do the same for `heading`:
```
link:
  font_family: 'does not exist'

heading:
  font_family: 'does not exist'
```

I am using using asciidoctorj-pdf 1.5.0-alpha.11. Does the fact that it's AsciidoctorJ matter?

**Comment by mojavelinux** [comment]
That's correct. This is a limitation at the moment. It's not currently possible to change the font family, font size, or font style.

**Comment by mojavelinux** [comment]
This issue was resolved as part of the fix for #567.
